src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -102,10 +106,13 @@ function transitionTrackHeaders(
     oncoprint: OncoprintJS
 ) {
     if (
-        !_.isEqual(nextProps.heatmapTrackHeaders, prevProps.heatmapTrackHeaders)
+        !_.isEqual(
```

Comment:
i know this is old and venerable logic but it would help to have a comment explaining what "track transition" is in layman's terms.